src/api/models.rs

```diff
@@ -0,0 +1,45 @@
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone)]
```

Comment:
```suggestion
#[derive(Serialize, Deserialize, Clone, Debug)]
```
Would be nice to log them for debugging.